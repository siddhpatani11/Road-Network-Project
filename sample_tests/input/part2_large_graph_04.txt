139 477
0 23 19 G
0 29 6 R
0 64 15 B
1 13 7 B
1 32 4 G
1 38 19 R
1 53 9 G
1 78 50 G
1 92 43 G
1 99 27 G
2 77 34 R
2 82 3 R
3 9 21 R
3 24 45 G
4 41 48 B
4 52 41 B
4 57 5 R
4 131 16 R
5 57 35 B
5 77 17 B
6 84 40 R
6 103 16 R
6 114 28 G
6 127 2 G
7 54 37 G
7 65 30 R
7 105 46 B
7 135 38 G
8 29 15 G
8 69 7 R
9 29 20 B
9 35 39 G
9 87 50 R
10 63 38 R
10 68 50 R
10 74 28 B
10 112 20 B
10 113 30 B
11 5 20 B
11 29 36 R
11 38 36 R
11 42 30 G
11 47 8 B
11 81 4 R
11 114 21 R
11 126 15 R
12 34 16 G
12 73 50 R
12 126 27 B
13 8 42 B
13 86 25 G
13 94 39 G
14 18 10 R
14 69 38 G
14 89 38 G
14 93 2 R
14 101 21 B
14 137 31 B
15 89 48 G
15 128 19 G
15 136 22 G
16 42 23 R
16 111 35 G
16 134 51 R
17 28 7 G
17 136 2 B
18 33 38 R
18 124 4 B
18 127 12 R
19 77 23 B
19 123 20 G
20 0 40 B
20 65 35 B
20 107 22 R
21 53 46 G
22 132 6 R
23 42 24 G
23 114 40 R
23 124 16 R
23 135 44 R
24 78 48 R
25 57 14 B
25 77 42 G
25 82 12 R
25 97 24 B
25 131 46 R
26 17 37 R
26 31 7 B
27 25 3 R
27 44 17 G
27 126 12 G
28 15 48 B
28 51 5 B
28 58 37 G
28 102 20 B
29 1 32 R
29 38 35 B
29 98 46 B
29 123 17 G
29 137 43 B
30 0 32 G
30 69 39 R
30 83 18 R
30 118 17 G
31 98 37 R
31 120 18 B
32 34 36 R
32 54 44 B
32 130 39 G
33 49 16 R
33 64 16 G
33 65 34 B
33 66 26 R
33 90 31 R
33 124 9 G
33 126 9 B
33 135 19 R
33 137 45 G
34 43 24 B
34 98 36 B
35 27 45 R
35 50 40 R
35 105 27 B
36 6 22 G
36 14 51 G
36 20 30 B
36 35 3 G
36 72 51 B
37 27 26 B
37 34 44 B
37 41 32 R
37 116 45 G
38 79 45 G
38 86 13 R
39 15 3 B
39 21 43 G
39 88 15 B
40 52 14 B
40 77 49 B
40 79 46 R
40 102 2 B
41 44 12 R
42 30 41 R
42 71 37 G
43 44 16 B
43 87 21 B
44 73 44 G
44 110 5 B
45 91 19 B
45 107 19 B
45 126 25 G
46 51 13 B
46 77 9 B
46 83 4 G
47 25 19 R
47 63 31 G
47 66 11 B
47 138 12 R
48 34 47 R
48 110 29 G
49 44 20 B
49 58 4 R
49 59 3 B
49 95 22 B
50 4 5 B
50 9 24 G
50 85 34 B
51 53 12 G
51 89 2 R
51 105 44 R
52 34 41 R
52 56 29 R
52 57 20 G
52 137 2 G
53 54 31 G
54 18 35 B
54 123 17 G
54 132 47 B
55 5 1 B
55 72 30 B
55 123 16 R
56 123 28 R
56 132 15 B
57 107 37 G
57 131 21 B
58 21 35 B
59 24 19 G
59 61 28 R
59 69 43 B
59 73 20 B
59 115 4 R
59 132 40 G
60 16 46 B
60 72 7 G
60 86 6 G
61 70 25 B
61 89 39 B
61 99 30 R
62 66 45 B
62 133 19 B
63 65 26 R
64 46 13 G
64 110 45 R
65 98 14 B
65 107 46 R
66 4 20 G
66 29 16 R
66 33 2 B
66 54 9 R
66 87 7 G
66 107 42 G
66 116 32 G
66 118 14 G
66 128 46 G
67 6 22 R
67 13 29 B
67 43 21 G
67 52 44 B
67 79 17 B
68 0 35 B
68 40 5 G
68 60 48 R
68 77 20 G
68 109 43 G
69 45 49 G
69 50 41 R
69 104 4 B
70 58 26 R
70 66 38 R
70 81 38 B
70 104 26 B
71 12 6 G
71 24 20 G
71 64 15 G
71 114 50 B
71 127 20 B
71 138 34 B
72 46 18 B
72 73 46 R
72 95 11 G
72 96 10 R
72 131 42 G
73 62 18 G
73 127 29 B
74 11 2 B
74 23 8 B
74 106 19 G
74 108 22 R
75 77 8 B
75 126 36 R
76 21 30 R
76 41 19 G
76 60 35 R
76 71 19 B
77 4 3 R
77 14 50 B
77 36 4 R
77 106 16 B
77 131 22 G
78 73 19 R
78 117 15 R
79 13 45 G
79 63 49 R
79 138 33 R
80 38 19 B
80 85 5 B
80 115 30 R
80 119 30 B
81 65 4 R
81 72 19 G
81 127 42 G
82 49 45 G
82 57 20 R
82 120 11 B
82 136 49 G
83 69 18 G
83 91 48 R
84 1 27 B
84 33 34 R
84 113 36 B
84 120 28 G
85 24 18 G
85 30 14 B
85 76 6 B
85 80 28 R
85 124 25 B
86 27 1 R
86 32 20 B
86 58 25 G
86 69 7 R
87 9 18 G
87 58 20 R
88 34 49 B
88 43 1 G
89 21 31 B
89 32 9 B
89 52 43 B
89 59 21 G
89 102 25 G
89 106 34 R
89 127 18 B
90 46 4 R
90 88 43 G
90 106 20 R
90 109 15 G
90 127 41 B
91 5 14 G
91 38 37 R
91 47 48 B
91 50 19 B
91 123 32 R
92 1 15 B
92 48 28 G
92 56 25 B
92 79 7 B
92 80 36 B
92 82 7 G
92 114 42 R
92 133 43 G
93 94 26 R
95 130 11 G
96 126 9 G
97 52 19 R
97 128 8 R
99 16 2 R
99 111 1 G
99 127 35 R
100 9 18 R
100 52 7 G
100 54 19 G
100 61 9 B
100 66 51 B
100 97 32 G
100 110 46 B
101 48 12 G
102 19 31 G
102 31 38 B
102 60 30 R
102 62 47 G
102 65 37 B
103 19 50 B
103 57 1 R
103 64 24 R
103 82 24 B
103 101 9 R
103 111 44 B
103 131 39 B
104 32 38 R
104 43 43 R
104 69 8 R
104 78 29 G
104 127 49 R
105 8 13 R
105 25 33 R
105 117 25 G
106 54 39 R
106 135 15 R
107 7 51 G
107 17 45 R
107 117 5 R
107 133 24 B
108 54 16 B
108 58 7 R
108 130 20 B
109 8 3 B
110 48 48 R
110 85 39 B
110 134 29 B
111 8 40 R
111 12 15 G
111 121 45 B
112 8 38 G
112 33 30 R
112 60 21 B
112 73 14 B
112 95 27 G
113 30 26 B
113 49 50 B
113 107 30 B
113 114 32 G
113 134 21 B
114 7 46 B
114 22 31 G
114 34 4 R
114 50 23 B
114 66 10 R
114 94 49 R
114 95 29 R
114 97 49 R
114 130 33 B
114 133 11 G
115 19 12 R
115 62 16 R
116 45 7 R
116 68 38 B
117 30 25 G
117 56 17 G
117 82 9 R
117 128 27 R
117 136 20 B
118 32 5 B
118 41 46 G
118 59 33 G
119 9 18 G
119 23 7 G
119 25 6 B
119 63 35 G
119 132 8 B
120 78 48 R
120 97 45 B
120 118 37 G
120 119 21 G
120 124 40 G
121 17 17 G
121 36 44 B
121 117 2 B
121 123 2 R
122 48 39 B
124 61 31 B
125 13 42 G
125 21 16 G
125 33 19 B
125 55 39 R
125 84 3 R
125 91 13 R
125 106 23 B
125 111 36 G
125 114 6 G
126 26 12 R
126 27 39 B
126 61 34 B
126 105 51 G
126 109 18 R
126 116 22 G
127 40 30 B
127 85 13 R
128 13 36 G
128 18 29 B
128 92 39 G
129 22 33 G
129 26 12 B
129 32 16 G
129 82 22 B
130 1 34 R
130 25 5 G
131 7 3 R
131 33 47 G
132 50 50 B
132 70 8 B
132 72 18 R
132 76 47 G
132 98 17 B
132 129 21 R
133 11 11 B
133 16 19 R
133 50 21 B
133 99 20 G
133 111 32 B
134 34 44 B
134 74 30 B
134 101 7 G
134 102 3 B
134 104 36 B
135 44 39 B
135 90 17 G
135 118 46 G
135 132 47 B
136 91 25 B
136 97 30 R
136 122 35 R
136 123 18 R
136 129 40 G
137 94 39 G
137 100 10 R
137 119 47 R
138 13 36 G
138 39 44 B
