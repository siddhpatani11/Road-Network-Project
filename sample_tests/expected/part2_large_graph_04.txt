7 - -1 B 234 - -1 - -1 - -1 - -1 - -1 - 0 B 59 B 182 - -1 - -1 - -1 B 353 B 171 B 570 - -1 B 515 G 72 - -1 - -1 B 182 G 138 B 171 B 241 B 79 - -1 - -1 B 522 B 74 B 500 B 592 G 121 G 110 B 239 - -1 - -1 - -1 B 109 - -1 - -1 - -1 - -1 - -1 G 77 B 451 B 597 - -1 - -1 - -1 B 634 B 527 B 134 B 243 G 37 - -1 B 302 - -1 B 147 B 550 B 572 G 150 - -1 B 199 G 126 R 30 - -1 - -1 - -1 B 402 - -1 - -1 G 102 B 570 - -1 - -1 - -1 B 121 B 284 B 284 B 313 - -1 - -1 - -1 - -1 B 223 B 122 - -1 - -1 B 529 B 169 B 470 B 277 B 173 - -1 G 113 - -1 B 159 B 120 G 180 B 127 - -1 B 542 - -1 - -1 B 46 B 137 B 470 - -1 - -1 - -1 G 181 - -1 - -1 - -1 B 343 - -1 B 475 B 198 B 164 - -1 G 226 B 651 B 193 G 119 - -1 - -1 G 84 - -1 G 105 - -1 G 144 G 84 - -1 - -1 G 38 B 616 B 117 B 317 
